### PR TITLE
LPS-99304 adding the google file like a draft so when publish can be tagged

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

File: modules/apps/asset/asset-auto-tagger-test/src/testIntegration/java/com/liferay/asset/auto/tagger/test/BaseAssetAutoTaggerTestCase.java [hunk 1]
```diff
@@ -63,6 +65,8 @@
 	public void setUp() throws Exception {
 		ServiceTestUtil.setUser(TestPropsValues.getUser());
 
+		company = CompanyTestUtil.addCompany();
```
Comment (cgoncas) [inline modules/apps/asset/asset-auto-tagger-test/src/testIntegration/java/com/liferay/asset/auto/tagger/test/BaseAssetAutoTaggerTestCase.java]: Do you really need to Add a Company?

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by cgoncas** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/76936. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**